src/Components/Admin/Admin.tsx

```diff
@@ -143,6 +158,17 @@ const Admin = () => {
                     handleSubmit={handleSubmitEdit}
                 />
             </Modal>
+            <DeleteConfirmationModal
+                open={showDeleteModal}
+                onClose={() => {
+                    handleClose(setShowDeleteModal);
+                    setDeleteRoomId(null);
```

Comment:
Why have 2 functions here? Combine everything in one function ( outside of JSX ) and pass it as a prop if you need